### ISSUE TITLE
Temporary Files are not Removed

### ISSUE THREAD

**Issue by kelleyl** [clamsproject/universal-viewer-visualizer#5]
Currently the iiif manifest that is passed to the universal viewer is stored using a named temporary file. The plan was to allow the file to be deleted after the app is done using it, but with how the current code is structured it was necessary to set the delete parameter to false. This means that the files stick around in the data directory after the app is closed. Im currently just manually deleting them, but the code should be refactored so that the files will be deleted automatically after the app is closed.

**Comment by marcverhagen** [comment]
Yeah, good one. The same needs to happen for the mmif-visualizer and its WebVTT files, so maybe we should table this till our two visualizers are unified.
